mongo/integration/unified/client_entity.go

```diff
@@ -133,21 +176,171 @@ func (c *clientEntity) failedEvents() []*event.CommandFailedEvent {
 	return events
 }
 
+func getCurrentTime() int64 {
```

Comment:
nit: Rename to something like `getCurrentTimeSeconds()` or `getSecondsSinceEpoch` to document the unit of the return value.